
Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-credential-initializer-7jvhz[0m
{"level":"info","ts":1595254597.4403515,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1595254597.440921,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-working-dir-initializer-q9wll[0m
{"level":"info","ts":1595254598.2370334,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254598.2393339,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-create-dir-workspace-7hcjv[0m
{"level":"info","ts":1595254604.8673384,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254604.8692784,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-git-source-seldonio-seldon-core-pr-2169-li-9fw8m-62rtj[0m
{"level":"info","ts":1595254605.2159076,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1595254610.3188384,"logger":"fallback-logger","caller":"git/git.go:103","msg":"Successfully cloned https://github.com/SeldonIO/seldon-core.git @ master in path /workspace/source"}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-git-merge[0m
Using SHAs from PULL_REFS=master:683c23532db0abcd2d764a138c00952fb6c57fc6,2169:995d5f58e6225ab16b713f2a69f8fdf8a63dbd16
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git fetch origin --unshallow 995d5f58e6225ab16b713f2a69f8fdf8a63dbd16: 683c23532db0abcd2d764a138c00952fb6c57fc6:
DEBUG: ran git fetch --unshallow origin 995d5f58e6225ab16b713f2a69f8fdf8a63dbd16: 683c23532db0abcd2d764a138c00952fb6c57fc6: in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git branch
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git checkout master
DEBUG: ran git checkout master in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git reset --hard 683c23532db0abcd2d764a138c00952fb6c57fc6
DEBUG: ran git reset --hard 683c23532db0abcd2d764a138c00952fb6c57fc6 in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git clean -fd .
DEBUG: ran clean --force -d . in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git merge 995d5f58e6225ab16b713f2a69f8fdf8a63dbd16
DEBUG: ran git merge 995d5f58e6225ab16b713f2a69f8fdf8a63dbd16 in 
DEBUG: LC_ALL=C JX_LOG_LEVEL=error git log --format=%H%x1f%an%x1f%ae%x1f%cn%x1f%ce%x1f%s%n%b%x1e 683c23532db0abcd2d764a138c00952fb6c57fc6..HEAD
Merged SHA c457118574cc965d1e9b054d193c211db5f5bef0 with commit message 'Merge commit '995d5f58e6225ab16b713f2a69f8fdf8a63dbd16'' into base branch master
Merged SHA 995d5f58e6225ab16b713f2a69f8fdf8a63dbd16 with commit message 'upgrade k8s.io and jaeger-client-go libraries' into base branch master

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=2169
+ BUILD_NUMBER=2
+ PIPELINE_CONTEXT=lint
++ cat
+ COMMENT_MSG='## STARTING TEST [ lint # 2 ]

Logs will be available when test completes in the following link:
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2169/2.log

Impatient try:
** get build logs SeldonIO/seldon-core/PR-2169 --build=2'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=2169 '--comment=## STARTING TEST [ lint # 2 ]

Logs will be available when test completes in the following link:
https://github.com/SeldonIO/seldon-core/blob/gh-pages/jenkins-x/logs/SeldonIO/seldon-core/PR-2169/2.log

Impatient try:
** get build logs SeldonIO/seldon-core/PR-2169 --build=2'

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-source-mkdir-seldonio-seldon-core-pr-2169-li-9fw8m-qlmwj[0m
{"level":"info","ts":1595254632.4162927,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254632.4215405,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment[0m and container [32mstep-source-copy-seldonio-seldon-core-pr-2169-li-9fw8m-dwlp8[0m
{"level":"info","ts":1595254632.7038388,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254633.7417123,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r source/. /pvc/pr-build-comment/workspace\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-credential-initializer-hpr67[0m
{"level":"info","ts":1595255016.589341,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1595255016.5906105,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-working-dir-initializer-mftbp[0m
{"level":"info","ts":1595255017.7474751,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255017.9052732,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-create-dir-workspace-fpcnw[0m
{"level":"info","ts":1595255024.7921891,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255024.7940035,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-source-copy-workspace-sx7dg[0m
{"level":"info","ts":1595255025.240417,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255032.643936,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-operator[0m and container [32mstep-lint-operator[0m
make: Entering directory '/workspace/source/operator'
# Check if licenses have changed
git \
	--no-pager diff \
	--exit-code \
	./licenses
make: Leaving directory '/workspace/source/operator'

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-credential-initializer-w64qf[0m
{"level":"info","ts":1595254652.1185415,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1595254652.119048,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-working-dir-initializer-ktdsb[0m
{"level":"info","ts":1595254652.8575256,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254652.8596058,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-create-dir-workspace-sxcvs[0m
{"level":"info","ts":1595254659.352643,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254659.3541677,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-source-copy-workspace-jhnpw[0m
{"level":"info","ts":1595254659.849708,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595254669.9801388,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-python[0m and container [32mstep-lint-python[0m
make: Entering directory '/workspace/source/python'
pip install -e . -r requirements-dev.txt
Obtaining file:///workspace/source/python
Requirement already satisfied: Flask<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 1)) (1.1.2)
Requirement already satisfied: jsonschema<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 2)) (3.2.0)
Requirement already satisfied: Flask-cors<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 3)) (3.0.8)
Requirement already satisfied: redis<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 4)) (3.5.3)
Requirement already satisfied: requests<3.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 5)) (2.24.0)
Requirement already satisfied: numpy<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 6)) (1.19.0)
Requirement already satisfied: flatbuffers<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 7)) (1.12)
Requirement already satisfied: protobuf<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 8)) (3.12.2)
Requirement already satisfied: grpcio<2.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 9)) (1.30.0)
Requirement already satisfied: Flask-OpenTracing<1.2.0,>=1.1.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 10)) (1.1.0)
Requirement already satisfied: opentracing<2.3.0,>=2.2.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 11)) (2.2.0)
Requirement already satisfied: jaeger-client<4.2.0,>=4.1.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 12)) (4.1.0)
Requirement already satisfied: grpcio-opentracing<1.2.0,>=1.1.4 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 13)) (1.1.4)
Requirement already satisfied: PyYAML<5.4 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 14)) (5.3.1)
Requirement already satisfied: gunicorn<20.1.0,>=19.9.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 15)) (20.0.4)
Requirement already satisfied: minio<6.0.0,>=4.0.9 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 16)) (5.0.10)
Requirement already satisfied: azure-storage-blob<3.0.0,>=2.0.1 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 17)) (2.1.0)
Requirement already satisfied: setuptools>=41.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 18)) (49.2.0)
Requirement already satisfied: google-cloud-storage>=1.16.0 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 19)) (1.29.0)
Requirement already satisfied: prometheus_client<0.9.0,>=0.7.1 in /usr/local/lib/python3.6/site-packages (from -r requirements.txt (line 20)) (0.8.0)
Requirement already satisfied: black==19.10b0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 2)) (19.10b0)
Requirement already satisfied: flake8==3.8.1 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 3)) (3.8.1)
Requirement already satisfied: mypy<0.783 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 4)) (0.782)
Requirement already satisfied: Pillow==7.2.0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 5)) (7.2.0)
Requirement already satisfied: pytest==5.4.2 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 6)) (5.4.2)
Requirement already satisfied: pytest-cov==2.10.0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 7)) (2.10.0)
Requirement already satisfied: tox<4.0.0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 8)) (3.17.1)
Requirement already satisfied: grpcio-tools==1.30.0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 9)) (1.30.0)
Requirement already satisfied: mypy-protobuf==1.22 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 10)) (1.22)
Requirement already satisfied: tenacity==6.2.0 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 13)) (6.2.0)
Requirement already satisfied: coverage==5.2 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 16)) (5.2)
Requirement already satisfied: pandas==1.0.1 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 19)) (1.0.1)
Requirement already satisfied: pip-licenses==2.2.1 in /usr/local/lib/python3.6/site-packages (from -r requirements-dev.txt (line 22)) (2.2.1)
Requirement already satisfied: click>=5.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (7.1.2)
Requirement already satisfied: Werkzeug>=0.15 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (1.0.1)
Requirement already satisfied: itsdangerous>=0.24 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (1.1.0)
Requirement already satisfied: Jinja2>=2.10.1 in /usr/local/lib/python3.6/site-packages (from Flask<2.0.0->-r requirements.txt (line 1)) (2.11.2)
Requirement already satisfied: pyrsistent>=0.14.0 in /usr/local/lib/python3.6/site-packages (from jsonschema<4.0.0->-r requirements.txt (line 2)) (0.16.0)
Requirement already satisfied: six>=1.11.0 in /usr/local/lib/python3.6/site-packages (from jsonschema<4.0.0->-r requirements.txt (line 2)) (1.15.0)
Requirement already satisfied: importlib-metadata; python_version < "3.8" in /usr/local/lib/python3.6/site-packages (from jsonschema<4.0.0->-r requirements.txt (line 2)) (1.7.0)
Requirement already satisfied: attrs>=17.4.0 in /usr/local/lib/python3.6/site-packages (from jsonschema<4.0.0->-r requirements.txt (line 2)) (19.3.0)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 5)) (2020.6.20)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 5)) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 5)) (2.10)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/site-packages (from requests<3.0.0->-r requirements.txt (line 5)) (1.25.9)
Requirement already satisfied: tornado<6,>=4.3 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->-r requirements.txt (line 12)) (5.1.1)
Requirement already satisfied: threadloop<2,>=1 in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->-r requirements.txt (line 12)) (1.0.2)
Requirement already satisfied: thrift in /usr/local/lib/python3.6/site-packages (from jaeger-client<4.2.0,>=4.1.0->-r requirements.txt (line 12)) (0.13.0)
Requirement already satisfied: python-dateutil in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 16)) (2.8.1)
Requirement already satisfied: pytz in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 16)) (2020.1)
Requirement already satisfied: configparser in /usr/local/lib/python3.6/site-packages (from minio<6.0.0,>=4.0.9->-r requirements.txt (line 16)) (5.0.0)
Requirement already satisfied: azure-storage-common~=2.1 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->-r requirements.txt (line 17)) (2.1.0)
Requirement already satisfied: azure-common>=1.1.5 in /usr/local/lib/python3.6/site-packages (from azure-storage-blob<3.0.0,>=2.0.1->-r requirements.txt (line 17)) (1.1.25)
Requirement already satisfied: google-auth<2.0dev,>=1.11.0 in /usr/local/lib/python3.6/site-packages (from google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (1.19.2)
Requirement already satisfied: google-cloud-core<2.0dev,>=1.2.0 in /usr/local/lib/python3.6/site-packages (from google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (1.3.0)
Requirement already satisfied: google-resumable-media<0.6dev,>=0.5.0 in /usr/local/lib/python3.6/site-packages (from google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (0.5.1)
Requirement already satisfied: typed-ast>=1.4.0 in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (1.4.1)
Requirement already satisfied: regex in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (2020.7.14)
Requirement already satisfied: toml>=0.9.4 in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (0.10.1)
Requirement already satisfied: appdirs in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (1.4.4)
Requirement already satisfied: pathspec<1,>=0.6 in /usr/local/lib/python3.6/site-packages (from black==19.10b0->-r requirements-dev.txt (line 2)) (0.8.0)
Requirement already satisfied: pycodestyle<2.7.0,>=2.6.0a1 in /usr/local/lib/python3.6/site-packages (from flake8==3.8.1->-r requirements-dev.txt (line 3)) (2.6.0)
Requirement already satisfied: pyflakes<2.3.0,>=2.2.0 in /usr/local/lib/python3.6/site-packages (from flake8==3.8.1->-r requirements-dev.txt (line 3)) (2.2.0)
Requirement already satisfied: mccabe<0.7.0,>=0.6.0 in /usr/local/lib/python3.6/site-packages (from flake8==3.8.1->-r requirements-dev.txt (line 3)) (0.6.1)
Requirement already satisfied: typing-extensions>=3.7.4 in /usr/local/lib/python3.6/site-packages (from mypy<0.783->-r requirements-dev.txt (line 4)) (3.7.4.2)
Requirement already satisfied: mypy-extensions<0.5.0,>=0.4.3 in /usr/local/lib/python3.6/site-packages (from mypy<0.783->-r requirements-dev.txt (line 4)) (0.4.3)
Requirement already satisfied: packaging in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (20.4)
Requirement already satisfied: more-itertools>=4.0.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (8.4.0)
Requirement already satisfied: pluggy<1.0,>=0.12 in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (0.13.1)
Requirement already satisfied: wcwidth in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (0.2.5)
Requirement already satisfied: py>=1.5.0 in /usr/local/lib/python3.6/site-packages (from pytest==5.4.2->-r requirements-dev.txt (line 6)) (1.9.0)
Requirement already satisfied: virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0 in /usr/local/lib/python3.6/site-packages (from tox<4.0.0->-r requirements-dev.txt (line 8)) (20.0.27)
Requirement already satisfied: filelock>=3.0.0 in /usr/local/lib/python3.6/site-packages (from tox<4.0.0->-r requirements-dev.txt (line 8)) (3.0.12)
Requirement already satisfied: PTable in /usr/local/lib/python3.6/site-packages (from pip-licenses==2.2.1->-r requirements-dev.txt (line 22)) (0.9.2)
Requirement already satisfied: MarkupSafe>=0.23 in /usr/local/lib/python3.6/site-packages (from Jinja2>=2.10.1->Flask<2.0.0->-r requirements.txt (line 1)) (1.1.1)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/site-packages (from importlib-metadata; python_version < "3.8"->jsonschema<4.0.0->-r requirements.txt (line 2)) (3.1.0)
Requirement already satisfied: cryptography in /usr/local/lib/python3.6/site-packages (from azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->-r requirements.txt (line 17)) (2.9.2)
Requirement already satisfied: pyasn1-modules>=0.2.1 in /usr/local/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.11.0->google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (0.2.8)
Requirement already satisfied: rsa<5,>=3.1.4; python_version >= "3" in /usr/local/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.11.0->google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (4.6)
Requirement already satisfied: cachetools<5.0,>=2.0.0 in /usr/local/lib/python3.6/site-packages (from google-auth<2.0dev,>=1.11.0->google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (4.1.1)
Requirement already satisfied: google-api-core<2.0.0dev,>=1.16.0 in /usr/local/lib/python3.6/site-packages (from google-cloud-core<2.0dev,>=1.2.0->google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (1.21.0)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/site-packages (from packaging->pytest==5.4.2->-r requirements-dev.txt (line 6)) (2.4.7)
Requirement already satisfied: importlib-resources>=1.0; python_version < "3.7" in /usr/local/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox<4.0.0->-r requirements-dev.txt (line 8)) (3.0.0)
Requirement already satisfied: distlib<1,>=0.3.1 in /usr/local/lib/python3.6/site-packages (from virtualenv!=20.0.0,!=20.0.1,!=20.0.2,!=20.0.3,!=20.0.4,!=20.0.5,!=20.0.6,!=20.0.7,>=16.0.0->tox<4.0.0->-r requirements-dev.txt (line 8)) (0.3.1)
Requirement already satisfied: cffi!=1.11.3,>=1.8 in /usr/local/lib/python3.6/site-packages (from cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->-r requirements.txt (line 17)) (1.14.0)
Requirement already satisfied: pyasn1<0.5.0,>=0.4.6 in /usr/local/lib/python3.6/site-packages (from pyasn1-modules>=0.2.1->google-auth<2.0dev,>=1.11.0->google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (0.4.8)
Requirement already satisfied: googleapis-common-protos<2.0dev,>=1.6.0 in /usr/local/lib/python3.6/site-packages (from google-api-core<2.0.0dev,>=1.16.0->google-cloud-core<2.0dev,>=1.2.0->google-cloud-storage>=1.16.0->-r requirements.txt (line 19)) (1.52.0)
Requirement already satisfied: pycparser in /usr/local/lib/python3.6/site-packages (from cffi!=1.11.3,>=1.8->cryptography->azure-storage-common~=2.1->azure-storage-blob<3.0.0,>=2.0.1->-r requirements.txt (line 17)) (2.20)
Installing collected packages: seldon-core
  Running setup.py develop for seldon-core
Successfully installed seldon-core
tox --recreate -e licenses
GLOB sdist-make: /workspace/source/python/setup.py
licenses create: /workspace/source/python/.tox/licenses
licenses installdeps: pip-licenses
licenses inst: /workspace/source/python/.tox/.tmp/package/1/seldon-core-1.2.2.dev0.zip
licenses installed: absl-py==0.9.0,astunparse==1.6.3,attrs==19.3.0,azure-common==1.1.25,azure-storage-blob==2.1.0,azure-storage-common==2.1.0,cachetools==4.1.1,certifi==2020.6.20,cffi==1.14.0,chardet==3.0.4,click==7.1.2,configparser==5.0.0,cryptography==2.9.2,Flask==1.1.2,Flask-Cors==3.0.8,Flask-OpenTracing==1.1.0,flatbuffers==1.12,gast==0.3.3,google-api-core==1.21.0,google-auth==1.19.2,google-auth-oauthlib==0.4.1,google-cloud-core==1.3.0,google-cloud-storage==1.29.0,google-pasta==0.2.0,google-resumable-media==0.5.1,googleapis-common-protos==1.52.0,grpcio==1.30.0,grpcio-opentracing==1.1.4,gunicorn==20.0.4,h5py==2.10.0,idna==2.10,importlib-metadata==1.7.0,itsdangerous==1.1.0,jaeger-client==4.1.0,Jinja2==2.11.2,jsonschema==3.2.0,Keras-Preprocessing==1.1.2,Markdown==3.2.2,MarkupSafe==1.1.1,minio==5.0.10,numpy==1.19.0,oauthlib==3.1.0,opentracing==2.2.0,opt-einsum==3.3.0,pip-licenses==2.2.1,prometheus-client==0.8.0,protobuf==3.12.2,PTable==0.9.2,pyasn1==0.4.8,pyasn1-modules==0.2.8,pycparser==2.20,pyrsistent==0.16.0,python-dateutil==2.8.1,pytz==2020.1,PyYAML==5.3.1,redis==3.5.3,requests==2.24.0,requests-oauthlib==1.3.0,rsa==4.6,scipy==1.4.1,seldon-core @ file:///workspace/source/python/.tox/.tmp/package/1/seldon-core-1.2.2.dev0.zip,six==1.15.0,tensorboard==2.2.2,tensorboard-plugin-wit==1.7.0,tensorflow==2.2.0,tensorflow-estimator==2.2.0,termcolor==1.1.0,threadloop==1.0.2,thrift==0.13.0,tornado==5.1.1,urllib3==1.25.9,Werkzeug==1.0.1,wrapt==1.12.1,zipp==3.1.0
licenses run-test-pre: PYTHONHASHSEED='3883845689'
licenses run-test: commands[0] | pip-licenses --from=mixed --format=csv --output-file=./licenses/license_info.csv
created path: ./licenses/license_info.csv
licenses run-test: commands[1] | pip-licenses --from=mixed --format=plain-vertical --with-license-file --no-license-path --output-file=./licenses/license.txt
created path: ./licenses/license.txt
___________________________________ summary ____________________________________
  licenses: commands succeeded
  congratulations :)
cut -d, -f1,3 ./licenses/license_info.csv \
	> ./licenses/license_info.no_versions.csv
black \
	--check ./ ../testing \
	--exclude "(testing/scripts/proto|seldon_core/proto/|.eggs|.tox)"
All done! ✨ 🍰 ✨
75 files would be left unchanged.
flake8
0
# Check if licenses have changed
git \
	--no-pager diff \
	--exit-code \
	./licenses/license_info.no_versions.csv
make: Leaving directory '/workspace/source/python'

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-credential-initializer-sbnk2[0m
{"level":"info","ts":1595255016.589575,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1595255016.5901742,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-working-dir-initializer-xzkgr[0m
{"level":"info","ts":1595255017.7475114,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255017.9052923,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-create-dir-workspace-8d9fm[0m
{"level":"info","ts":1595255024.7662563,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255024.7691286,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-source-copy-workspace-tj887[0m
{"level":"info","ts":1595255025.3272421,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255032.6439717,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mlint-executor[0m and container [32mstep-lint-executor[0m
make: Entering directory '/workspace/source/executor'
# Check if licenses have changed
git \
	--no-pager diff \
	--exit-code \
	./licenses
make: Leaving directory '/workspace/source/executor'

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-credential-initializer-lcmpj[0m
{"level":"info","ts":1595255041.1639981,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/refs/heads/0.8.0-ns-support: no such file or directory"}
{"level":"info","ts":1595255041.1647134,"logger":"fallback-logger","caller":"creds-init/main.go:40","msg":"Credentials initialized."}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-working-dir-initializer-fmzck[0m
{"level":"info","ts":1595255042.0016932,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255042.0043218,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-place-tools[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-create-dir-workspace-m5c6h[0m
{"level":"info","ts":1595255050.522851,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255050.5248325,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c mkdir -p /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-source-copy-workspace-fggrr[0m
{"level":"info","ts":1595255050.865085,"logger":"fallback-logger","caller":"logging/config.go:69","msg":"Fetch GitHub commit ID from kodata failed: open /var/run/ko/HEAD: no such file or directory"}
{"level":"info","ts":1595255051.2689972,"logger":"fallback-logger","caller":"bash/main.go:64","msg":"Successfully executed command \"sh -c cp -r /pvc/pr-build-comment/workspace/. /workspace/source\"; output "}

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-setup-builder-home[0m

Showing logs for build [32mseldonio-seldon-core-pr-2169-li-9fw8m-2[0m stage [32mpr-build-comment-success[0m and container [32mstep-step2[0m
+++ dirname ./add-pr-build-comment-success
++ cd .
++ pwd
+ STARTUP_DIR=/workspace/source/ci
+ REPO_OWNER=SeldonIO
+ REPO_NAME=seldon-core
+ PULL_NUMBER=2169
+ BUILD_NUMBER=2
+ PIPELINE_CONTEXT=lint
++ cat
+ COMMENT_MSG='## SUCCESS [lint # 2] TEST PASSED ✅

Well done! 😎'
+ ** step pr comment --owner=SeldonIO --repository=seldon-core --pull-request=2169 '--comment=## SUCCESS [lint # 2] TEST PASSED ✅

Well done! 😎'
